### DISCUSSION TITLE
Tutorials for performing aerostructural optimization of high aspect ratio wings

### DISCUSSION THREAD

**Discussion by Aming12-01** [mdolab/dafoam#670] [Tutorials]
I am currently working on the aerostructural optimization of high aspect ratio wings. Given that these wings must be treated differently from low aspect ratio wings due to their nonlinear characteristics, I am interested in finding tutorials and scripts specifically designed to handle these types of problems. Any resources or guidance on how to approach the unique challenges posed by high aspect ratio wings would be greatly appreciated.

**Comment by friedenhe** [comment]
This is more related to the TACS setup to consider geometrical nonlinearity. Please contact the first author of this paper: https://arc.aiaa.org/doi/10.2514/6.2023-3316

**Reply by Aming12-01** [reply]
thank you professor
